src/types.d.ts

```diff
@@ -0,0 +1,15 @@
+export interface TypeObjTask {
+  tasks: {
+    id: number;
+    checked: boolean;
+    title: string;
+  };
+  onChangeTask: (task: Task) => void;
+}
+
+export interface Tasks {
```

Comment:
Aqui tu poderia criar um type Task no singular e quando for usar, use o tipo como array Task[]

Reply:
Essa aqui eu nao entendi muito bem